bundles/AdminBundle/Resources/public/js/pimcore/settings/email/log.js

```diff
@@ -349,6 +349,37 @@ pimcore.settings.email.log = Class.create({
                     }
                 ]
             },
+            {
+                xtype:'actioncolumn',
+                width: 30,
+                menuText: t('email_log_forward'),
+                items:[
+                    {
+                        tooltip: t('email_log_forward'),
+                        icon: '/bundles/pimcoreadmin/img/flat-color-icons/email-forward.svg',
+                        handler: function (grid, rowIndex) {
+                            var rec = grid.getStore().getAt(rowIndex);
+
+                            Ext.Ajax.request({
+                                url: '/admin/email/show-email-log?id=' + rec.get('id') + '&type=details',
+                                success: function(response){
+                                    var data = Ext.decode( response.responseText );
+                                    var win = this.getForwardEmailWindow(data);
+                                    win.show();
+                                }.bind(this),
+                                failure: function () {
+                                    alert('Could not foward email');
```

Comment:
alert ?

Reply:
@dvesh3 can you change that to a proper ExtJS alert box? Thanks!

Reply:
@brusch done 😉